pinc/ProjectTransition.inc

```diff
@@ -1144,11 +1145,26 @@ new ProjectTransition(
     PROJ_POST_SECOND_AVAILABLE,
     'holder',
     array(
-        'project_restriction'   => null,
+        'project_restriction'   => "project_was_not_returned_from_PPV",
+        'action_name'           => _("Upload for Verification"),
+        'confirmation_question' => null,
+        'detour'                => "$code_url/tools/upload_text.php?project=<PROJECTID>&stage=post_1",
+        'settings_template'     => "modifieddate='<TIMESTAMP>', postproofer=checkedoutby, ppverifier=NULL, postcomments=CONCAT(postcomments,'<G:postcomments>')",
+        'collateral_actions'    => 'PP_to_PPV_collateral',
+        'disabled_during_SR'    => true,
+    )
+);
+
+new ProjectTransition(
+    PROJ_POST_FIRST_CHECKED_OUT,
+    PROJ_POST_SECOND_AVAILABLE,
+    'proj_facilitator',
```

Comment:
I'll change it thus.

Reply:
I'm happy to leave it to holders and Squirrels as well. It shouldn't occur that often and if there is a serious need later for PFs to have it we'll reconsider later

Reply:
The issue where there was some disagreement in the task centre was 'when a project has been returned from the PPVer to the PPer for further work, should the PPer be able to return it to the PPV pool as well as back to the same PPVer?'

Reply:
The one wanting the "Upload for Verification" button hidden from PPers when the PPer returns a project to PPV was okrick. Both windymilla and I had concerns about the appropriateness of denying the PPer the ability to return the project straight to the PPV pool instead of to the previous PPVer, but if that's the way it is going to be implemented, the decision of when an override is appropriate should be left to the squirrels in consultation with the PPer and possibly PPVer.

Reply:
It can be implemented in any way. Your wish is my command. Otherwise I might be influenced by what is easiest (or most challenging) to code:)